pkgs/top-level/python-packages.nix

```diff
@@ -2456,6 +2458,8 @@ in {
 
   google_cloud_monitoring = callPackage ../development/python-modules/google_cloud_monitoring { };
 
+  google_cloud_org_policy = callPackage ../development/python-modules/google_cloud_org_policy { };
```

Comment:
Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)

Reply:
So to be clear, change `google_cloud_org_policy = callPackage ../development/python-modules/google_cloud_org_policy { };` to `google-cloud-org-policy = callPackage ../development/python-modules/google_cloud_org_policy { };`?

Reply:
the directory name should match:
```
google-cloud-org-policy = callPackage ../development/python-modules/google-cloud-org-policy { };
```

the `pname` inside the expression can differ

Reply:
Fixed.